src/Logger.hs

```diff
@@ -20,7 +20,5 @@ data LogEntry =
   | LogEntryForModerator EventLogItemCold
   deriving (Eq, Show)
 
-type SendLogMsg = LogEntry -> IO ()
```

Comment:
one more reason: it has become much less widely used to other code improvements.